packages/node-resolve/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Rich Harris <richard.a.harris@gmail.com>",
   "homepage": "https://github.com/rollup/plugins/tree/master/packages/node-resolve/#readme",
   "bugs": "https://github.com/rollup/plugins/issues",
-  "main": "dist/index.js",
+  "main": "./dist/cjs/index.js",
```

Comment:
do we need `./`?

Reply:
This is a requirement in "exports" I just changed main and module for consistency.